### DISCUSSION TITLE
Explain values in conf.py

### DISCUSSION THREAD

**Discussion by QueryType** [okuvshynov/slowllama#11] [Q&A] [answered]
Is it possible to get some explanation and suggestions on the values defined in conf.py? I am not much aware of the torch apis, so some examples and suggestions would be really nice to get started on custom data. Or if there is some resource already available can have a look there too. Thanks in advance.

**Comment by okuvshynov** [comment, marked as answer]
Let me add some comments in the code itself. thank you

**Reply by okuvshynov** [reply]
added comments in https://github.com/okuvshynov/slowllama/commit/cffad8c619283d8a0d4bcd11fd25ea1d1e3c88fe
